includes/multi-currency/class-wc-payments-multi-currency.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * Class WC_Payments_Multi_Currency
+ *
+ * @package WooCommerce\Payments
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class that controls Multi Currency functionality.
+ */
+class WC_Payments_Multi_Currency {
+
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var WC_Payments_Multi_Currency
+	 */
+	protected static $instance = null;
+
+	/**
+	 * The available currencies.
+	 *
+	 * @var array
+	 */
+	public $available_currencies;
```

Comment:
Perhaps we should make this and `$default_currency` and `$enabled_currencies``protected` rather than `public` so that they can only be modified within the context of the `WC_Payments_Multi_Currency` class. We can always add helper methods to help us get/set the values of these from outside the class if that is needed. I see you've already added `get_` functions for these, which is great.

Reply:
Updated the props here to be protected in https://github.com/Automattic/woocommerce-payments/pull/1795/commits/cdb969523e396de384ea946505bdcbb3a4b144b9